gk/main.c

```diff
@@ -93,3 +109,22 @@ run_gk(const struct gk_config *gk_conf)
 
 	return 0;
 }
+
+int
+cleanup_gk(void)
+{
+	/*
+	 * Atomically decrements the atomic counter (v) by one and returns true 
+	 * if the result is 0, or false in all other cases.
+	 */
+	if (rte_atomic32_dec_and_test(&gk_conf->ref_cnt) == true) {
```

Comment:
The `== true` is redundant, the return of `rte_atomic32_dec_and_test()` is already logic. Drop `== true`; the code will do the same thing.